lib/mumukit/login/provider/auth0.rb

```diff
@@ -10,6 +10,7 @@ def configure_omniauth!(omniauth)
   def request_authentication!(controller, login_settings)
     settings = lock_settings(controller, login_settings, {closable: false})
     controller.render_html! <<HTML
+ #{header_html}
```

Comment:
Has it sense to have the header both in `request_authentication!` **and** `header_html`? Perhaps we could leave the header empty and directly embed this JS here.